brage-import/src/test/java/no/sikt/nva/brage/migration/lambda/BrageEntryEventConsumerTest.java

```diff
@@ -1114,50 +1113,6 @@ void shouldMergePublicationContextAndUseCourseFromBragePublicationWhenPublicatio
         assertThat(((UnconfirmedCourse) actualPublicationContext.getCourse()).code(), is(equalTo(subjectCode)));
     }
 
-    @Test
-    void shouldKeepOrderOfRelatedDocumentsWhenMergingDegreePhd()
```

Comment:
Not relevant longer, we set sequence number instead of sorting values in a list.